governance/nested-governance-processing.php

```diff
@@ -81,49 +81,67 @@ public static function get_nested_settings_and_css( $governance_rules ) {
 	 * @param array $nodes             The metadata of the nodes that have been built so far.
 	 * @param array $current_selector  The current selector of the current block.
 	 * @param array $current_path      The current path to the block.
-	 * 
-	 * @return array 
+	 *
+	 * @return array
 	 */
 	private static function get_settings_of_blocks( $blocks_registered, $current_block, $nodes = [], $current_selector = null, $current_path = [] ) {
 		foreach ( $current_block as $block_name => $block ) {
-			if ( array_key_exists( $block_name, $blocks_registered ) ) {
-
-				$selector = is_null( $current_selector ) ? null : $current_selector;
+			if ( ! self::is_block_supported( $block_name, $blocks_registered ) ) {
+				continue;
+			}
 
-				// This function is only available in 6.3 and above.
-				if ( function_exists( ( 'wp_get_block_css_selector' ) ) ) {
-					$looked_up_selector = wp_get_block_css_selector( $blocks_registered[ $block_name ] );
-				} else {
-					// Once the 6.3 upgrade is done, this will be deprecated.
-					$looked_up_selector = self::get_css_selector_for_block( $block_name, $blocks_registered );
-				}
+			$selector = is_null( $current_selector ) ? null : $current_selector;
+
+			// If the block name ends with /* or is just *, then it's a wildcard rule and we need to use the wp-block selector to match against any block.
+			if ( str_ends_with( $block_name, '/*' ) || ( '*' === $block_name ) ) {
+				// Due to the fact that the paragraph block doesn't have a wp-block-paragraph class, we need to add it manually.
+				// In addition, wp-block is prefixed to the block name so this allows us to target all blocks.
+				$looked_up_selector = 'p, [class*=wp-block]';
```

Comment:
This is the fix I put in for the issue you found. I realized paragraphs have a similar bug so this targets them as well.

Reply:
Glad you were able to find a fix!